mlflow/server/js/src/components/LoginPage.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+
+class LoginPage extends Component {
+  render() {
+    return (
+        <div>
+            <a href="http://127.0.0.1:5000/login/authorize"> Login with Github. </a>
```

Comment:
maybe should use Mlflow port env variable (it should it exists), it is smth that gets defined when creating mlflow server (and can vary). In k8s, thinking of having port 80 instead, for example..